src/scss/cdb-components/forms/dropdowns.scss

```diff
@@ -0,0 +1,119 @@
+// Dropdown styles
+// ----------------------------------------------
+
+/* SG
+# Forms/Dropdowns
+
+```
+<div class="CDB-DropdownContainer">
+  <input type="text" name="text" placeholder="DejaVu Sans" class="CDB-InputText CDB-Text">
+  <div class="CDB-Dropdown CDB-Box-Modal">
+    <div class="CDB-Dropdown-calculations CDB-Text is-semibold">
+      <ul>
+        <li class="CDB-Dropdown-calculationsElement u-iBlock">
+          <input class="CDB-Radio" type="radio" name="calculation" value="01" checked>
+          <span class="u-iBlock CDB-Radio-face"></span>
+          <label class="u-iBlock u-lSpace">COUNT</label>
+        </li>
+        <li class="CDB-Dropdown-calculationsElement u-iBlock">
+          <input class="CDB-Radio" type="radio" name="calculation" value="02">
+          <span class="u-iBlock CDB-Radio-face"></span>
+          <label class="u-iBlock u-lSpace">SUM</label>
+        </li>
+        <li class="CDB-Dropdown-calculationsElement u-iBlock">
+          <input class="CDB-Radio" type="radio" name="calculation" value="03">
+          <span class="u-iBlock CDB-Radio-face"></span>
+          <label class="u-iBlock u-lSpace">AVG</label>
+        </li>
+        <li class="CDB-Dropdown-calculationsElement u-iBlock">
+          <input class="CDB-Radio" type="radio" name="calculation" value="04">
+          <span class="u-iBlock CDB-Radio-face"></span>
+          <label class="u-iBlock u-lSpace">MAX</label>
+        </li>
+        <li class="CDB-Dropdown-calculationsElement u-iBlock">
+          <input class="CDB-Radio" type="radio" name="calculation" value="05">
+          <span class="u-iBlock CDB-Radio-face"></span>
+          <label class="u-iBlock u-lSpace">MIN</label>
+        </li>
+      </ul>
+    </div>
+
+    <div class="CDB-Dropdown-options CDB-Text">
+      <ul>
+        <li class="CDB-Dropdown-optionsElement is-selected">
+          <span class="u-iBlock u-lSpace CDB-Size-medium">selected</span>
+        </li>
+        <li class="CDB-Dropdown-optionsElement">
+          <span class="u-iBlock u-lSpace CDB-Size-medium">active</span>
+        </li>
+        <li class="CDB-Dropdown-optionsElement is-disabled">
+          <span class="u-iBlock u-lSpace CDB-Size-medium">disabled</span>
+        </li>
+        <li class="CDB-Dropdown-optionsElement is-disabled">
+          <span class="u-iBlock u-lSpace CDB-Size-medium">disabled</span>
+        </li>
+      </ul>
+    </div>
+  </div>
+</div>
+```
+*/
+
+@import '../../cdb-utilities/mixins';
+@import '../../cdb-variables/sizes';
+@import '../../cdb-variables/colors';
+
+.CDB-DropdownContainer {
+  position: relative;
+  height: 250px;
+}
+.CDB-Dropdown {
+  @include display-flex();
+  @include flex-direction(row);
+  position: absolute;
+  top: 40px;
+  height: 200px;
```

Comment:
IMO this height is no necessary, we should add a max-height to 'Dropdown-options'